deepmd/dpmodel/model/base_model.py

```diff
@@ -151,38 +157,6 @@ class BaseModel(BaseBaseModel):
         Backend-independent BaseModel class.
     """
 
-    __plugins = Plugin()
-
-    @staticmethod
-    def register(key: str) -> Callable[[object], object]:
-        """Register a descriptor plugin.
-
-        Parameters
-        ----------
-        key : str
-            the key of a descriptor
-
-        Returns
-        -------
-        callable[[object], object]
-            the registered descriptor
-
-        Examples
-        --------
-        >>> @Model.register("some_model")
-            class SomeModel(Model):
-                pass
-        """
-        return BaseModel.__plugins.register(key)
-
-    def __new__(cls, *args, **kwargs):
-        if cls is BaseModel:
-            cls = cls.get_class_by_type(j_get_type(kwargs, cls.__name__))
-        return super().__new__(cls)
-
-    @classmethod
-    def get_class_by_type(cls, model_type: str) -> Type["BaseModel"]:
-        if model_type in BaseModel.__plugins.plugins:
-            return BaseModel.__plugins.plugins[model_type]
-        else:
-            raise RuntimeError("Unknown model type: " + model_type)
+    def get_model_def_script(self) -> str:
```

Comment:
The method may be moved to make base atomic model, the same designed as the pt backend.